### ISSUE TITLE
500 Error Due to UserSchemaAttribute and UserSchemaAttribute items sending null values

### ISSUE THREAD

**Issue by rcollette** [okta/okta-sdk-dotnet#712]
### Describe the bug?

Both UserSchemaAttribute and UserSchemaAttributeItems apply annotations to property members to send default values, which is null for non intrinsics.  A 500 error results when calling the schema API due to the null values being present.

### What is expected to happen?

Null properties should not be sent in API calls like:

```bash
curl --location 'https://sso.precisely.services/api/v1/meta/schemas/apps/theSchemaId/default' \
--header 'Authorization: SSWS theToken' \
--header 'Content-Type: application/json' \
--header 'Cookie: JSESSIONID=theSessionId' \
--data '{
    "definitions": {
        "custom": {
            "id": "#custom",
            "properties": {
                "firstName": {
                    "scope": "NONE",
                    "type": "string",
                    "externalName": "firstName",
                    "maxLength": 50,
                    "minLength": 0,
                    "required": true,
                    "title": "firstName"
                },
                "lastName": {
                    "scope": "NONE",
                    "type": "string",
                    "externalName": "lastName",
                    "maxLength": 50,
                    "minLength": 0,
                    "required": true,
                    "title": "lastName"
                },
                "email": {
                    "scope": "NONE",
                    "type": "string",
                    "externalName": "email",
                    "maxLength": 100,
                    "minLength": 0,
                    "required": true,
                    "title": "email"
                },
                "mobilePhone": {
                    "scope": "NONE",
                    "type": "string",
                    "externalName": "mobilePhone",
                    "maxLength": 100,
                    "minLength": 0,
                    "required": false,
                    "title": "Mobile Phone"
                },
                "federatedGroups": {
                    "scope": "NONE",
                    "type": "array",
                    "union": "DISABLE",
                    "externalName": "federatedGroups",
                    "items": {
                        "type": "string"
                    },
                    "required": false,
                    "title": "federatedGroups"
                }
            },
            "required": [
                "firstName",
                "lastName",
                "email"
            ],
            "type": "object"
        }
    },
    "properties": {
        "profile": {
            "allOf": [
                null,
                null
            ]
        }
    },
    "title": "identity.precisely.services-00u208epblm6akcYu0h8 User"
}'
```

### What is the actual behavior?

Null values are being sent
```bash
curl --location 'https://sso.precisely.services/api/v1/meta/schemas/apps/theSchemaId/default' \
--header 'Authorization: SSWS theToken' \
--header 'Content-Type: application/json' \
--header 'Cookie: JSESSIONID=theSessionId' \
--data '{
    "definitions": {
        "custom": {
            "id": "#custom",
            "properties": {
                "firstName": {
                    "scope": "NONE",
                    "type": "string",
                    "union": null,
                    "description": null,
                    "enum": null,
                    "externalName": "firstName",
                    "externalNamespace": null,
                    "items": null,
                    "master": null,
                    "maxLength": 50,
                    "minLength": 0,
                    "mutability": null,
                    "oneOf": null,
                    "pattern": null,
                    "permissions": null,
                    "required": true,
                    "title": "firstName",
                    "unique": null
                },
                "lastName": {
                    "scope": "NONE",
                    "type": "string",
                    "union": null,
                    "description": null,
                    "enum": null,
                    "externalName": "lastName",
                    "externalNamespace": null,
                    "items": null,
                    "master": null,
                    "maxLength": 50,
                    "minLength": 0,
                    "mutability": null,
                    "oneOf": null,
                    "pattern": null,
                    "permissions": null,
                    "required": true,
                    "title": "lastName",
                    "unique": null
                },
                "email": {
                    "scope": "NONE",
                    "type": "string",
                    "union": null,
                    "description": null,
                    "enum": null,
                    "externalName": "email",
                    "externalNamespace": null,
                    "items": null,
                    "master": null,
                    "maxLength": 100,
                    "minLength": 0,
                    "mutability": null,
                    "oneOf": null,
                    "pattern": null,
                    "permissions": null,
                    "required": true,
                    "title": "email",
                    "unique": null
                },
                "mobilePhone": {
                    "scope": "NONE",
                    "type": "string",
                    "union": null,
                    "description": null,
                    "enum": null,
                    "externalName": "mobilePhone",
                    "externalNamespace": null,
                    "items": null,
                    "master": null,
                    "maxLength": 100,
                    "minLength": 0,
                    "mutability": null,
                    "oneOf": null,
                    "pattern": null,
                    "permissions": null,
                    "required": false,
                    "title": "Mobile Phone",
                    "unique": null
                },
                "federatedGroups": {
                    "scope": "NONE",
                    "type": "array",
                    "union": "DISABLE",
                    "externalName": "federatedGroups",
                    "items": {
                        "enum": null,
                        "oneOf": null,
                        "type": "string"
                    },
                    "required": false,
                    "title": "federatedGroups"
                }
            },
            "required": [
                "firstName",
                "lastName",
                "email"
            ],
            "type": "object"
        }
    },
    "properties": {
        "profile": {
            "allOf": [
                null,
                null
            ]
        }
    },
    "title": "IdpTitle User"
}'
```

### Reproduction Steps?

See https://github.com/okta/okta-sdk-dotnet/issues/702 for same reproduction steps.

### Additional Information?

See https://github.com/okta/okta-sdk-dotnet/issues/702 for workaround code.

### .NET Version

8.0.203

### SDK Version

7.0.6

**Comment by bryanapellanes-okta** [comment]
@rcollette Thanks for bringing this to our attention.  I've created an internal issue for tracking and am investigating furhter to determine an appropriate fix. (OKTA-727760)

**Comment by bryanapellanes-okta** [comment]
@rcollette Thanks for your patience while we investigate this issue.  I have a clear understanding of the problem that you've described, however, I have so far not been able to reproduce the 500 error in an integration test written in C# code.  I was able to reproduce the problem described in #702 and that issue is resolved by the change in #713 .  Is there a specific Api call that causes a 500 error due to the emission of null defaults?  May I have your assistance to reproduce the issue with a C# example?
